### ISSUE TITLE
Comply with Wikimedia User-Agent policy

### ISSUE THREAD

**Issue by zstadler** [IsraelHikingMap/Site#1573]
## What I expect to happen
The daily update should be able to download Wikimedia images

## What really happened
Access to the Wikimedia images receives a Forbidden (403) error

## Things I think the developers should know, images, links etc.
From the site log:

```
error: (403) Forbidden. Please comply with the User-Agent policy: https://meta.wikimedia.org/wiki/User-Agent_policy.
2021-08-16 05:33:21.1023 | 14 |WARN | There was a problem with the following image url: https://upload.wikimedia.org/wikipedia/he/b/b8/Yeshivat_maale_giboa.jpg  The remote server returned an error: (403) Forbidden. Please comply with the User-Agent policy: https://meta.wikimedia.org/wiki/User-Agent_policy.
```

**Comment by HarelM** [comment]
Actually user-agent is set to `IsraelHikingMapSite/5.x` so I don't know what to do here.
Closing this for now.
If there's another issue we'll reopen...

**Comment by zstadler** [comment]
As of 5-Sep-21, the site continues to receive these errors from Wikimedia.

It looks like the current User-Agent value does not have contact information. Please try this value:

    User-Agent: IsraelHikingMapSite/5.x (https://israelhiking.osm.org.il; israelhikingmap@gmail.com)

The [Wikimedia User-Agent policy](https://meta.wikimedia.org/wiki/User-Agent_policy) says:

> If you run a bot, please send a User-Agent header identifying the bot with an identifier that isn't going to be confused with many other bots, and supplying some way of contacting you (e.g. a userpage on the local wiki, a userpage on a related wiki using interwiki linking syntax, a URI for a relevant external website, or an email address), e.g.:
> 
>     User-Agent: CoolTool/0.0 (https://example.org/cool-tool/; cool-tool@example.org) generic-library/0.0
> 
> The generic format is `<client name>/<version> (<contact information>) <library/framework name>/<version> [<library name>/<version> ...]`. Parts that are not applicable can be omitted.
> 
> If you run an automated agent, please consider following the Internet-wide convention of including the string "bot" in the User-Agent string, in any combination of lowercase or uppercase letters. This is recognized by Wikimedia's systems, and used to classify traffic and provide more accurate statistics. 

Analyzing and testing this issue from a arbitrary IP is probably irrelevant, because it has not been downloading much Wikimedia content while the server's IP was probably flagged as abusing by Wikimedia.

**Comment by HarelM** [comment]
I have no objections to change this, although I have my doubts it will solve the issue.
Feel free to send a PR with the relevant fix. I guess we should add the word "bot" somewhere in there...

**Comment by HarelM** [comment]
Solved by #1597
